shortcuts/o.yml

```diff
@@ -5016,6 +5010,11 @@ km 1:
   include:
   - key: km 1
     namespace: pka
+kn 0:
+  url: https://www.kleinanzeigen.de/m-nachrichten.html
+  title: Kleinanzeigen, Nachrichten
+  tags:
+  - shopping
```

Comment:
😦 This will not work, as there is already a `kn` keyword.

Should we then just keep `ekn`?